lib/view/json/JsonLijstResponse.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require_once 'view/JsonResponse.class.php';
+
+/**
+ * JsonLijstResponse.class.php
+ *
+ * @author G.J.W. Oolbekkink <g.j.w.oolbekkink@gmail.com>
+ *
+ */
+class JsonLijstResponse extends JsonResponse {
+    public function view() {
+        http_response_code($this->code);
+        header('Content-Type: application/json');
+
+        echo "[\n";
+        $response = array();
+        foreach ($this->model as $entity) {
+            $response[] = $this->getJson($entity);
```

Comment:
Het probleem hier is dat je naast je objecten in het geheugen nu ook de serialized versie in het geheugen laad en daarna NOG een keer voor de implode en dan pas echo-ed. Zie DataTableResponse hoe ik dat probleem heb omzeild met een flag voor wanneer er een komma tussen moet.